doc/source/dev/development_environment.rst

```diff
@@ -83,6 +83,7 @@ Before running the tests, first install the test dependencies::
 
     $ python -m pip install -r requirements/test_requirements.txt
     $ python -m pip install asv # only for running benchmarks
+    $ python -m pip install spin # utility for running the tests
```

Comment:
This would not be needed anymore with the change in `test_requirements.txt`, right?

Reply:
Agreed, this change should be undone.